packages/logs/src/domain/assembly.spec.ts

```diff
@@ -43,10 +43,13 @@ const COMMON_CONTEXT_WITH_USER: CommonContext = {
 
 describe('startLogsAssembly', () => {
   const sessionManager: LogsSessionManager = {
-    findTrackedSession: (_startTime, options) =>
-      (sessionIsActive && sessionIsTracked) || options?.returnInactive
-        ? { id: sessionIsTracked ? SESSION_ID : undefined }
-        : undefined,
```

Comment:
This was not correct. According to the [implementation](https://github.com/DataDog/browser-sdk/blob/ed08cb2d6a7a05adf7789ab89e02b94e27c917f2/packages/logs/src/domain/logsSessionManager.ts#L32C5-L39C7), if a session is not tracked, it should return `undefined` regardless of `returnInactive` option